### PR TITLE
Add parameterized way to create/set pool and task

### PR DESCRIPTION
**Pull request by xingwu1**
The content to be added to Changelog is as follows:
* 
*  Add parameterized way to create/update batch entities
*

### CODE REVIEW

File: lib/commands/batch/batch.pool._js [hunk 1]
```diff
@@ -60,7 +62,21 @@ exports.init = function(cli) {
     
   pool.command('create [json-file]')
     .description($('Create a Batch pool'))
-    .option('-f, --json-file <json-file>', $('the file containing the pool object to create in JSON format'))
+    .option('-f, --json-file <json-file>', $('the file containing the pool object to create in JSON format, if this parameter is specified, all other pool parameters are ignored'))
+    .option('-i, --id <pool-id>', $('the Batch pool id'))
+    .option('-s, --vm-size <vm-size>', $('the size of virtual machines in the pool, ex: small, Standard_D14'))
```
Comment (amarzavery) [inline lib/commands/batch/batch.pool._js]: -s is reserved for subscription

Comment (xingwu1) [inline lib/commands/batch/batch.pool._js]: We don't use subscription parameter for these data plane commands.

Comment (amarzavery) [inline lib/commands/batch/batch.pool._js]: I see. However, we still treat -s as a special case for subscription and want to make sure that this is not used by any command apart from using it for subscription. That keeps things simple.

Comment (xingwu1) [inline lib/commands/batch/batch.pool._js]: Fixed.

**Pull request merged**